### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by katerynashylina**
- [DEMO LINK](https://katerynashylina.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://katerynashylina.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1 +1,62 @@
...
+  flex-wrap: wrap;
+  padding: 10px 0;
+}
+
+.list {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  list-style: none;
+  line-height: 60px;
```
Comment (SanyaBratashchuk) [inline src/style.css:31]:
```suggestion
```
Use vertical paddings for links instead

File: src/index.html [hunk 2]
```diff
@@ -7,8 +7,62 @@
...
+        src="images/logo.png"
+        alt="moyo_logo"
+        class="logo"
+        >
+      </a>
+
+      <nav class="nav">
+        
+        <ul class="list">
+          <li class="nav__list">
```
Comment (SanyaBratashchuk) [inline src/index.html:27]:
```suggestion
        <ul class="nav__list">
          <li class="nav__item">
```

File: src/index.html [hunk 3]
```diff
@@ -7,8 +7,62 @@
...
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="http://localhost:8080/">
+        <img 
+        src="images/logo.png"
+        alt="moyo_logo"
+        class="logo"
+        >
```
Comment (SanyaBratashchuk) [inline src/index.html:21]:
```suggestion
        <img 
          src="images/logo.png"
          alt="moyo_logo"
          class="logo"
        >
```

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 4]
```diff
@@ -7,8 +7,62 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,700;1,400;1,700&display=swap" rel="stylesheet">
```
Comment (SanyaBratashchuk) [inline src/index.html:12]: You didn't connect font-weight: 500. That's why you can't pass a tests

File: src/style.css (hunk 1 above)
Comment (SanyaBratashchuk) [reply src/style.css:31]: image
Didn't fixed

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]

**Review by liliia-livandovskaya — CHANGES_REQUESTED** [review]
Pay attention to the tests, for example, LAPTOPS & COMPUTERS should be blue, but you have black. Please fix it 

Screenshot 2022-12-08 at 17 10 58

File: src/index.html [hunk 5]
```diff
@@ -1,14 +1,71 @@
...
+          <li class="nav__item">
+            <a href="#video" class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
   </body>
-</html>
+
+</html>
```
Comment (SanyaBratashchuk) [inline src/index.html:71]:
```suggestion
</html>

```

File: src/style.css [hunk 6]
```diff
@@ -1 +1,62 @@
...
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 10px 0;
+}
+
+.nav__list {
```
Comment (SanyaBratashchuk) [inline src/style.css:26]:
```suggestion
.nav__list {
padding: 0; 
```

File: src/style.css [hunk 7]
```diff
@@ -1 +1,62 @@
...
+  align-items: center;
+  background-color: #fff;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 10px 0;
```
Comment (SanyaBratashchuk) [inline src/style.css:23]:
```suggestion
```

File: src/style.css [hunk 8]
```diff
@@ -1 +1,62 @@
...
+}
+
+.nav__list {
+  margin: 0;
+  line-height: 60px;
```
Comment (SanyaBratashchuk) [inline src/style.css:28]: do not hardcode height use vertical paddings for links instead

File: src/style.css [hunk 9]
```diff
@@ -1 +1,62 @@
-/* Styles go here */
+html {
+  font-size: 12px;
+  font-family: Roboto, sans-serif;
+  font-weight: 500px;
```
Comment (SanyaBratashchuk) [inline src/style.css:4]:
```suggestion
  font-weight: 500;
```

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 10]
```diff
@@ -1,14 +1,70 @@
...
+        <img 
+          src="images/logo.png"
+          alt="moyo_logo"
+          class="logo"
+        >
+      </a>
+
+      <nav class="nav">
+
+        <ul class="nav__list">
```
Comment (ivanchuk-yurii) [inline src/index.html:28]:
```suggestion
      <nav class="nav">
        <ul class="nav__list">
```

File: src/style.css [hunk 11]
```diff
@@ -1 +1,61 @@
...
+  color: #000;
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.is-active,
+.laptop,
+.nav__item:hover {
```
Comment (ivanchuk-yurii) [inline src/style.css:47]:
```suggestion
.is-active,
.nav__item:hover {
```
this shouldn't be highlighted by default. move hover from `li` to `a` instead
image

File: src/style.css [hunk 12]
```diff
@@ -1 +1,61 @@
...
+  color: #00acdc;
+}
+
+.nav__link:active::after,
+.is-active::after {
+  content: "";
+  position: absolute;
+  top: 32px;
```
Comment (ivanchuk-yurii) [inline src/style.css:55]:
```suggestion
  bottom: 0;
```

File: src/style.css [hunk 13]
```diff
@@ -1 +1,61 @@
...
+  margin-right: 20px;
+}
+
+.is-active,
+.laptop,
+.nav__item:hover {
+  color: #00acdc;
+}
+
+.nav__link:active::after,
```
Comment (ivanchuk-yurii) [inline src/style.css:51]: you don't need this

File: src/style.css [hunk 14]
```diff
@@ -1 +1,61 @@
...
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.is-active,
+.laptop,
+.nav__item:hover {
+  color: #00acdc;
```
Comment (ivanchuk-yurii) [inline src/style.css:48]: move color into variable

**Review by ivanchuk-yurii — CHANGES_REQUESTED** [review]

**Pull request closed without merging**